include/envoy/buffer/buffer.h

```diff
@@ -14,8 +14,8 @@ namespace Buffer {
  * A raw memory data slice including location and length.
  */
 struct RawSlice {
-  void* mem_;
-  size_t len_;
+  void* mem_ = nullptr;
```

Comment:
Nit: there is a slight preference for writing `mem_{}` etc. in various places in the Envoy code base, but it doesn't really matter, and we're not fully consistent.